app/views/users/show_follow.html.erb

```diff
@@ -1,19 +1,19 @@
-<% provide(:title, @title) %>
+<% provide(:title, @presenter.title) %>
 
-<% cache [@user, @title] do %>
+<% cache @presenter.cache_key do %>
 <div class="row">
   <aside class="span4">
     <section>
-      <%= gravatar_for @user %>
-      <h1><%= @user.name %></h1>
-      <span><%= link_to "view my profile", @user %></span>
-      <span><b>Microposts:</b> <%= @user.microposts.count %></span>
+      <%= gravatar_for @presenter.user %>
+      <h1><%= @presenter.user.name %></h1>
```

Comment:
You can delegate specific methods to the user object. E.g. `user_name` would delegate the `name` method to the user. The key thing would be decouple the view with knowing how the what to go through to get the user name. Allows the presenter could override this and provide implementation of `user_name` if necessary.